apps/ledger-live-mobile/src/components/CollapsibleList.tsx

```diff
@@ -148,11 +158,13 @@ const CollapsibleList = ({
         </View>
       </TouchableWithoutFeedback>
       <AnimatedFlatList
-        style={{
-          opacity: openingAnim,
-        }}
+        style={[
+          {
+            opacity: openingAnim,
+          },
+        ]}
         data={data}
-        renderItem={renderItem}
+        renderItem={renderItem as ListRenderItem<unknown>}
```

Comment:
the casting kinda makes the typing of the props useless, can be removed

Reply:
no, because `AnimatedFlatList` erases the generic type from `FlatList` which is unknown by default